packages/fxa-shared/email/helpers.js

```diff
@@ -0,0 +1,18 @@
+function normalizeEmail(originalEmail) {
+  return originalEmail.toLowerCase();
+}
+
+function emailsMatch(firstEmail, secondEmail) {
+  return (
+    firstEmail.localeCompare(secondEmail, undefined, {
+      // We use 'accent' instead of 'base' here to ensure a character with
+      // an accent is not considered the same as that character without one
+      sensitivity: 'accent',
+    }) === 0
+  );
+}
```

Comment:
FWIW, I would suggest getting the tests passing and landing this without changing the way the comparison is done, and moving to `localeCompare` in a followup PR. I wouldn't be surprised to find some subtle change in behaviour somewhere in the system that flows from the change here. (Although I don't think we have too many tests with unicode emails, so perhaps such bustage wouldn't show up in tests anyway).

Reply:
That's fair - Unless any pushback, I'll leave this in here, but commented out so we retain the knowledge for the future.

Reply:
Heh, I ordinarily wouldn't be...but personally I'm OK with leaving this in the code since it's such a small snippet; please consider also leaving a link to this discussion so future-us can easily rediscover the context.